src/csvcubed/utils/csvdataset.py

```diff
@@ -306,15 +306,28 @@ def _melt_data_set(
     ]
     id_cols = list(set(data_set.columns) - set(value_cols))
 
+    # Checking for any columns with the title "Value" and changing the value_name
+    # parameter passed to the melt function to "Not-Value" so that we don't
+    # trigger a pandas ValueError.
```

Comment:
I like the comment, should avoid confusion in the future if anyone else comes across this.

Reply:
Thanks, this is where code should be removed when addressing the [follow up ticket](https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gh/gss-cogs/csvcubed/860) #860 .